.github/workflows/pr-merged.yml

```diff
@@ -6,8 +6,8 @@ on:
       - main
 
 env:
-  AWS_REGION: us-east-1
-  REPOSITORY: 329710836760.dkr.ecr.us-east-1.amazonaws.com/rasa-sdk-dev
+  AWS_REGION: eu-west-1
+  REPOSITORY: 329710836760.dkr.ecr.us-east-1.amazonaws.com/rasa/rasa-sdk-dev
```

Comment:
Shouldnt this be `329710836760.dkr.ecr.eu-west-1.amazonaws.com/rasa/rasa-sdk-dev`?